manifests-servicemesh/1.0.0/servicemeshoperator.v1.0.0.clusterserviceversion.yaml

```diff
@@ -410,6 +415,6 @@ spec:
           - 'urn:alm:descriptor:com.tectonic.ui:booleanSwitch'
       - description: Limits describes the minimum/maximum amount of compute resources required/allowed
         displayName: Default Resource Requirements
-        path: istio.defaultResources
+        path: istio.global.defaultResources
```

Comment:
We should come back and expand these at a later point.

Reply:
Nods, although I don't think these are used anywhere in the web UI.